src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -21,6 +21,28 @@ public class LibGpiodDriver : UnixDriver
 
         private static string s_consumerName = System.Diagnostics.Process.GetCurrentProcess().ProcessName;
 
+        // for use the bias flags we need libgpiod version 1.5 or later
+        private static bool IsLibgpiodVersion1_5orHigher()
+        {
+            IntPtr libgpiodVersionPtr = Interop.libgpiod.gpiod_version_string();
```

Comment:
If we can validate that this method is also available in older versions of libgpiod that is perfect. If not, we could just add a try catch and in case the PInvoke is unsuccesful or doesn't bind we can just return false and assume it is not 1.5 or later.